File: VDS_Frontend/src/app/vacuna/Models/vacuna.dto.ts
import { Mascota } from "src/app/mascota/Models/mascota.dto";
import { Usuario } from "src/app/usuarios/Models/usuario.dto";

export interface Vacuna {
    id?: number;
    nombre: string; // Nombre de la vacuna
    laboratorio: string; // Laboratorio de la vacuna
    fecha: string;  // Fecha de administración de la vacuna
    mascotaId: number; // ID de la mascota asociada a la vacuna
    veterinarioId: number; // ID del veterinario que admiistra la vacuna
    consultaId: number; // ID de la consulta donde se administra la vacuna
}
  

File: VDS_Backend/src/main/java/com/uoc/tfm/vds_backend/vacuna/dto/VacunaDTO.java
package com.uoc.tfm.vds_backend.vacuna.dto;

import java.time.LocalDateTime;

import lombok.Data;

@Data
public class VacunaDTO {
    private Long id;
    private String nombre;      
    private String laboratorio;
    private LocalDateTime fecha;
    private Long mascotaId;
    private Long consultaId;
}
